src/core/linalg/src/sparse/4C_linalg_map.hpp

```diff
@@ -126,7 +120,9 @@ namespace Core::LinAlg
     int MinMyGID(void) const { return map_->MinMyGID(); }
 
     //! Access function for Epetra_Comm communicator.
-    const Epetra_Comm& Comm() const { return map_->Comm(); }
+    const Epetra_Comm& EpetraComm() const { return map_->Comm(); }
+
+    MPI_Comm Comm() const { return Core::Communication::unpack_epetra_comm(map_->Comm()); }
```

Comment:
@maxfirmbach This changes the naming compared to the Epetra_Map. Is this fine or should I rename the function returning the MPI_Comm?